wpilibjIntegrationTests/build.gradle

```diff
@@ -24,8 +24,11 @@ dependencies {
     compile 'com.googlecode.junit-toolbox:junit-toolbox:2.0'
     compile 'org.apache.ant:ant:1.9.4'
     compile 'org.apache.ant:ant-junit:1.9.4'
+    compile files(wpilibj.sourceSets.test.output.classesDir)
 }
 
+compileJava.dependsOn tasks.getByPath(':wpilibj:testClasses')
```

Comment:
@333fred I tried doing the same thing you did in lines 19/20 and it didn't work. Test output doesn't behave the same way as source files.